{{cookiecutter.repository_name}}/.gitignore

```diff
@@ -100,3 +100,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
```

Comment:
This file is Python project-specific gitignore. If you have editor-specific files, best to use a global gitignore: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
The usage of a global gitignore is OK for some scenarios, but that is beyond our control and something that very few of the users of this cookiecutter template will end up using. So we cannot rely on that to prevent having some things eventually added to our repositories.

Also, I disagree with your first statement. This is the overall project gitignore, but not necessary related to Python stuff only. So I see no harm on adding editor-specific things that we do not want added to our repositories here as well.

Actually, I would even open issue to extend this list and add other things, such as the project files from other editors like idea or pycharm, backup files, ipynb_checkpoints, etc.

Reply:
We can think it over, please remove it from this PR and we can consider it in a new one

Reply:
sounds good